WebRoot/css/zm.css

```diff
@@ -3906,15 +3906,43 @@ HTML>BODY .appt-selected .appt_allday_body
 	height:auto;
 }
 
-.ZmPrefZimletListView .DwtListView-Rows {
-	overflow:visible;
+.ZmPrefZimletDetail {
+	@PrefZimletDetail@
 }
 
-.ZmPrefZimletListView .Row {
-	overflow:hidden;
+.ZOptionsItemsListView {
+	@PanelBorder@
```

Comment:
better to use some descriptive property name and then assing @PanelBorder@ to that property